open-company/open-company-web
# Add section

card: https://trello.com/c/iRtWbDk6

To test:
- deploy on staging: you need to use it to make sure the js compilation doesn't brake anything
- check the ToC floating and it's scroll bar on a small screen
- try to add a section: for now it just shows a popover image
- try to remove a section: this actually works so if you make any change on staging reimport the data for that company
- check that the plus line appears between each section, before the first section, after the last section and in empty categories
- check that the pen is shown in the finances section and that it works as expected
